acp/acp_local_test.go

```diff
@@ -652,3 +654,46 @@ func Test_LocalACP_PersistentMemory_CheckDocAccess_TrueIfHaveAccessFalseIfNotErr
 	errClose = localACP.Close()
 	require.Nil(t, errClose)
 }
+
+func Test_LocalACP_InMemory_AddPolicy_InvalidCreatorIDReturnsError(t *testing.T) {
+	ctx := context.Background()
+	localACP := NewLocalACP()
+
+	localACP.Init(ctx, "")
+	err := localACP.Start(ctx)
+	require.Nil(t, err)
+
+	policyID, err := localACP.AddPolicy(
+		ctx,
+		invalidIdentity,
+		validPolicy,
+	)
+
+	require.ErrorIs(t, err, ErrInvalidActorID)
+	require.Empty(t, policyID)
+
+	err = localACP.Close()
+	require.NoError(t, err)
+}
```

Comment:
todo: Add a counterpart test after this test like: `Test_LocalACP_PersistentMemory_AddPolicy_InvalidCreatorIDReturnsError` to test the same with persistence.